inventory/src/ts/components/inventory-window.tsx

```diff
@@ -54,7 +65,7 @@ export class InventoryWindow extends React.Component<InventoryWindowProps, Inven
     const itemGroups: JSX.Element[] = [];
     this.state.itemGroups.forEach((group: ItemGroup, index: number) => {
       itemGroups.push((
-          <li className="inventory-item" key={'item-group' + index} onDoubleClick={this.useItem.bind(this, group) } onContextMenu={this.dropItem.bind(this, group )}>
+          <li className="inventory-item" key={'item-group' + index} onClick={this.simDblClick.bind(this, this.useItem, group) } onContextMenu={this.dropItem.bind(this, group )}>
```

Comment:
It's just a preference thing, but I prefer just using an inline function rather than a bind as typescript will bind this implicitly and it is clearer to understand what is happening as the bind thing is confusing for a lot of people.

``` onClick={() => this.simDblClick(this.useItem, group} ```